backend/src/middleware/index.js

```diff
@@ -11,7 +11,7 @@ import user from './userMiddleware'
 import includedFields from './includedFieldsMiddleware'
 import orderBy from './orderByMiddleware'
 import validation from './validation'
-import notifications from './notifications'
+import handleContentData from './handleHtmlContent/handleContentData'
```

Comment:
maybe `./handleHtmlContent/...` is the appropriate location for all the content analysis functions like ` extractMentionedUsers`

Reply:
But that’s all already in `./handleHtmlContent/…` ! @ulfgebhardt